tests/asconfig/target/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
   "scripts": {
-    "test": "node ../index.js"
+    "test": "node ../index.js --target && node ../index.js --showConfig --runtime none"
```

Comment:
Is the `--target` without a target intended?

Reply:
Just fixed it!